src/styles/components/SubmissionViewer.scss

```diff
@@ -89,3 +89,87 @@
     }
   }
 }
+
+.submission-modal-wrapper {
+  position: fixed;
+  top: 0;
+  right: 0;
+  height: 100vh;
+  width: 100%;
+  z-index: 100;
+  overflow: hidden;
+
+  &__top-bar {
```

Comment:
I started using BEM for CSS styling at gumroad and implemented it here just to keep things clear and easy to identify.  It was sort of frustrating at first for me but I think it pays off.  Let me know what you think.

If you haven't had to use it before, it's like so:
`block__element--modifier`

* Block can be a page and its composite elements, well, elements. (i.e. `submission-modal__subsection--italicized` or something like `button-component--primary`/`button-component--danger`)
* in case of reusable, isolted components, block is that component
* Modifiers are for states (invalid, empty, etc).
* Block name should be specific/verbose enough to avoid potential clashes with existing or future class names.
    * `checkout-page`is better than `checkout`.
    * `button-component` is better than `button` (which may be being used already).
* Element names don't have to be specific since they are scoped by block anyway.

Reply:
Cool, I'm somewhat familiar with BEM, seems like a good way to organize class names. Might do a refactor to follow BEM later.